docs/_docs/plugins.md

```diff
@@ -59,13 +64,14 @@ values of the gem names of the plugins you'd like to use. An example:
   </p>
 </div>
 
-In general, plugins you make will fall into one of five categories:
+In general, plugins you make will fall broadly into one of five categories:
 
 1. [Generators](#generators)
 2. [Converters](#converters)
 3. [Commands](#commands)
 4. [Tags](#tags)
 5. [Hooks](#hooks)
+6. [List of available plugins](#available-plugins)
```

Comment:
that makes sense... but where should the link go?
I added it as a sentence at first, but it had no link, maybe putting it back with the link makes it work? 
```
 1. [Generators](#generators)
 2. [Converters](#converters)
 3. [Commands](#commands)
 4. [Tags](#tags)
 5. [Hooks](#hooks)

See the bottom of the page for a [List of available plugins](#available-plugins).
```
/cc @DirtyF